sys/net/gnrc/link_layer/lorawan/gnrc_lorawan.c

```diff
@@ -132,6 +132,10 @@ static void _config_radio(gnrc_lorawan_t *mac, uint32_t channel_freq, uint8_t dr
 
 static void _configure_rx_window(gnrc_lorawan_t *mac, uint32_t channel_freq, uint8_t dr)
 {
+    if (channel_freq) {
+        DEBUG("gnrc_lorawan: RX2 Frequency: %luHz \n", channel_freq);
+    }
```

Comment:
```suggestion
    DEBUG("gnrc_lorawan: RX2 Frequency: %luHz \n", channel_freq);
```

The DEBUG macro already optimizes out the `channel_freq` variable